apps/api/package.json

```diff
@@ -83,7 +83,8 @@
     "slugify": "^1.4.6",
     "swagger-ui-express": "^4.4.0",
     "twilio": "^4.14.1",
-    "uuid": "^8.3.2"
+    "uuid": "^8.3.2",
+    "blake2":"^5.0.0"
```

Comment:
Did you realize you can use blake2 with the node crypto package?
https://nodejs.org/api/crypto.html
It uses openssl underneath the hood.

Reply:
You should be able to pass in "blake2b512" to crypto package.
<img width="483" alt="CleanShot 2023-10-11 at 18 42 09@2x" src="https://github.com/novuhq/novu/assets/43915749/fcd3c2f1-b4d8-4041-9a14-e362c7eb095c">

Reply:
done